samples/msal-angular-v2-samples/angular9-v2-sample-app/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     globals: {
         __PORT__: 4200,
         __STARTCMD__: "npm start -- --port 4200",
-        __TIMEOUT__: 45000
+        __TIMEOUT__: 90000
```

Comment:
Again doubling the timeout. For things that do timeout, doesn't this increase the time to run the entire test pass?